lib/compilers/hlsl.js

```diff
@@ -35,17 +42,79 @@ export class HLSLCompiler extends BaseCompiler {
         super(info, env);
 
         this.compiler.supportsIntel = false;
+
+        if (default_dxc === null && this.compiler.group === 'dxc') {
```

Comment:
I'd really like not to rely on specially-named groups - it will be surprising if/when we refactor. Can you add a configuration option for the "default compiler" if you need this instead please. Then one specific compiler can be picked instead of "the first compiler we see that happens to be in the `dxc` group", which might lead to weird cases

Reply:
Just so I understand, can you clarify what is meant by "configuration option" in this context?

Reply:
In the `etc/config/hlsl.amazon.properties` file, for each compiler that needs to know the path of another compiler:
```
compiler.compilerId.dxcCompilerPath=/path/to/compiler
```

And then, in the rga constructor, add a line like:
```
this.dxcCompiler=this.compilerProps('dxcCompilerPath', 'path/to/default/if/not/set');
```